cylc/flow/scripts/workflow_state.py

```diff
@@ -124,9 +124,12 @@ def connect(self):
 
     async def check(self):
         """Return True if desired workflow state achieved, else False"""
-        return self.checker.task_state_met(
-            self.args['task'], self.args['cycle'],
-            self.args['status'], self.args['message'])
+        if self.args['status']:
+            return self.checker.task_state_met(
+                self.args['task'], self.args['cycle'], self.args['status'])
+        elif self.args['message']:
```

Comment:
We should consider erroring if both `status` and `message` are defined, otherwise it may look as though this is supported. Or alternatively do both, kinda valid to poll for `succeeded` plus a custom output e.g. "succeeded with this outcome"?